### DISCUSSION TITLE
can you do order() or limit() after fts textsearch()?

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#18478] [Questions] [answered]
seems when i add order or limit after textsearch result, it will complain there is no order /limit function. is anyone able to order/sort results from fts?

```
 supabase.from('').select('').textSearch().order(col, { ascending: false })
```

**Comment by GaryAustin1** [comment, marked as answer]
```const result = await supabase1
          .from(myTable)
        .select().textSearch('content','cat').order('content', {ascending: false})
        ```
   Works and ascending/descending on the text column works.
   Tested now.
   
   In supabase-js, no Typescript.

**Reply by devhandler** [reply]
interesting. let me test again
